celery/canvas.py

```diff
@@ -162,33 +165,33 @@ def on_signature(self, sig, **headers) -> dict:
              Dict: headers to update.
          """
 
-    def on_chord_header_start(self, chord, **header) -> dict:
```

Comment:
This caused a very annoying bug where `headers` had a key named `chord` and this caused an exception to be raised.